buildSrc/src/main/groovy/csde.gradle

```diff
@@ -162,29 +183,77 @@ tasks.register('createKeystore') {
 }
 
 
-tasks.register('buildCPI') {
+tasks.register('buildCPIs') {
     group = pluginGroupName
-    dependsOn('build', 'createGroupPolicy', 'createKeystore')
+    // dependsOn('contracts:build', 'workflows:build', 'createGroupPolicy', 'createKeystore', 'getNotaryServerCPB')
+
+    //def dependsOnTasks = subprojects.collect {it.tasks.withType(org.gradle.api.tasks.GradleBuild)}
```

Comment:
is this and the other commented out code valid , should this be going into a release branch of a public repo ? As this is quite confusing for a dev which comes along trying to use this

Reply:
The whole repo is a bit messy at the moment, this PR is to attempt to get us a working base line to start refactoring it and tidying it all up.

even though its a public repo, its not a branch we expect people to be using yet.

Reply:
This weeks target is to get it all tidied up